src/data/cech.json

```
@@ -0,0 +1,11 @@
+[
+    {   
+        "userId": 1,
+        "id": 1,
+        "title": "Click here to learn more",
+        "body": "for better understanding"
+
+
+    }
+
+]
```

```suggestion
[
    {
       "userId": 1,
        "id": 1,
        "title": "Click here to learn more",
        "body": "for better understanding"
    }
]
```
Better to remove excess whitespace